.travis/deploy-acs.sh

```diff
@@ -0,0 +1,167 @@
+      continue
+    done
+  
+  if [ "$PODS_COUNTER" -ge "$PODS_COUNTER_MAX" ]; then
+    pod_status
+    echo "Pods did not start - exit 1"
+    exit 1
+  fi
+}
+
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: $namespace
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: $namespace:psp
+  namespace: $namespace
+rules:
+- apiGroups:
+  - policy
+  resourceNames:
+  - kube-system
+  resources:
+  - podsecuritypolicies
+  verbs:
+  - use
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: $namespace:psp:default
+  namespace: $namespace
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: $namespace:psp
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: $namespace
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: $namespace:psp:$release_name_ingress-nginx-ingress
+  namespace: $namespace
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: $namespace:psp
+subjects:
+- kind: ServiceAccount
+  name: $release_name_ingress-nginx-ingress
+  namespace: $namespace
+---
+$(kubectl create secret docker-registry quay-registry-secret --dry-run=client --docker-server=${DOCKER_REGISTRY} --docker-username=${DOCKER_REGISTRY_USERNAME} --docker-password=${DOCKER_REGISTRY_PASSWORD} -n $namespace -o yaml)
+EOF
+
+# install ingress
+helm upgrade --install $release_name_ingress stable/nginx-ingress \
+--set controller.scope.enabled=true \
+--set controller.scope.namespace=$namespace \
+--set rbac.create=true \
+--set controller.config."proxy-body-size"="100m" \
+--set controller.service.targetPorts.https=80 \
+--set controller.service.annotations."service\.beta\.kubernetes\.io/aws-load-balancer-backend-protocol"="http" \
+--set controller.service.annotations."service\.beta\.kubernetes\.io/aws-load-balancer-ssl-ports"="https" \
+--set controller.service.annotations."service\.beta\.kubernetes\.io/aws-load-balancer-ssl-cert"="${ACM_CERTIFICATE}" \
+--set controller.service.annotations."external-dns\.alpha\.kubernetes\.io/hostname"="$namespace.dev.alfresco.me" \
+--set controller.service.annotations."service\.beta\.kubernetes\.io/aws-load-balancer-ssl-negotiation-policy"="ELBSecurityPolicy-TLS-1-2-2017-01" \
+--set controller.service.annotations."service\.beta\.kubernetes\.io/aws-load-balancer-security-groups"="${AWS_SG}" \
+--set controller.publishService.enabled=true \
+--wait \
+--atomic \
+--namespace $namespace
+
+# install acs
+helm dep up helm/alfresco-content-services
+helm upgrade --install $release_name_acs helm/alfresco-content-services \
+--values=$values_file \
+--set externalPort="443" \
+--set externalProtocol="https" \
+--set externalHost="$namespace.dev.alfresco.me" \
+--set persistence.enabled=true \
+--set persistence.storageClass.enabled=true \
+--set persistence.storageClass.name="nfs-client" \
+--set global.alfrescoRegistryPullSecrets=quay-registry-secret \
```

Comment:
Do we not have to pass an EFS filesystem ID anymore?

Reply:
nope, the nfs client deployment is already set on the cluster. the deployments only specify the storage class.